tools/deviceGen/input/device_limits.json

```diff
@@ -0,0 +1,2347 @@
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating16BitUnsignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating16BitSignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating32BitUnsignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating32BitSignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating64BitUnsignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating64BitSignedAccelerated",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated",
+            "value": false
+        },
+        {
+            "groupComment": [
+                "or VK_EXT_texel_buffer_alignment:",
+                "[DO NOT EXPOSE]: the single texel alignments, let people just overalign"
+            ]
+        },
+        {
+            "type": "size_t",
+            "name": "storageTexelBufferOffsetAlignmentBytes",
+            "value": "std::numeric_limits<size_t>::max()",
+            "compareFlipped": true
+
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "storageTexelBufferOffsetSingleTexelAlignment",
+            "expose": false
+        },
+        {
+            "type": "size_t",
+            "name": "uniformTexelBufferOffsetAlignmentBytes",
+            "value": "std::numeric_limits<size_t>::max()",
+            "compareFlipped": true
+        },
```

Comment:
group properly,

also since you'll be using the `value` for the HLSL default traits' values, you should use `nbl::hlsl::numeric_limits<size_t>::max` (its available in C++ too!)

P.S. `size_t` might not be available in HLSL and may have to change to `uint64_t` instead

Reply:
the `size_t` stuff is done